### PR TITLE
Ts/jev/start

### PR DESCRIPTION
**Pull request by jeromeEliezer**
New modelize for user is start:
business User done.

### CODE REVIEW

**Review by RebeccaRamalho — COMMENTED** [review]
Good job

File: apiHospital/.gitignore [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-
+.DS_Store
 node_modules/
 build/
 tmp/
 temp/
-.env
\ No newline at end of file
+.env
+*DS_Store
```
Comment (RebeccaRamalho) [inline apiHospital/.gitignore]: why do you use a Ds_store?

Comment (jeromeEliezer) [reply apiHospital/.gitignore]: is tools from my macbook pro

**Review by nawalpro — APPROVED** [review]

**Review by RebeccaRamalho — APPROVED** [review]

**Comment by nawalpro** [comment]
change approved

**Pull request merged**